data/assets/default_keybindings.asset

```diff
@@ -0,0 +1,275 @@
+local propertyHelper = asset.require("util/property_helper")
+
+
+
+local ToggleNativeUi = {
+  Identifier = "os.ToggleNativeUi",
+  Name = "Show native GUI",
+  Command = propertyHelper.invert("Modules.ImGUI.Enabled"),
+  Documentation = "Shows or hides the native UI",
+  GuiPath = "/System/GUI",
+  IsLocal = true
+}
+
+local ToggleShutdown = {
+  Identifier = "os.ToggleShutdown",
+  Name = "Toggle shutdown",
+  Command = "openspace.toggleShutdown()",
+  Documentation = "Toggles the shutdown that will stop OpenSpace after a grace period. Press again to cancel the shutdown during this period",
+  GuiPath = "/System",
+  IsLocal = true
+}
+
+local TakeScreenshot = {
+  Identifier = "os.TakeScreenshot",
+  Name = "Take screenshot",
+  Command = "openspace.takeScreenshot()",
+  Documentation = "Saves the contents of the screen to a file in the ${SCREENSHOTS} directory",
+  GuiPath = "/System/Rendering",
+  IsLocal = true
+}
+
+local TogglePauseInterpolated = {
+  Identifier = "os.TogglePauseInterpolated",
+  Name = "Toggle pause (interpolate)",
+  Command = "openspace.time.pauseToggleViaKeyboard()",
+  Documentation = "Smoothly starts and stops the simulation time",
+  GuiPath = "/Time/Simulation Speed",
+  IsLocal = true
+}
+
+local TogglePauseImmediate = {
+  Identifier = "os.TogglePauseImmediate",
+  Name = "Toggle pause (immediate)",
+  Command = "openspace.time.togglePause()",
+  Documentation = "Immediately starts and stops the simulation time",
+  GuiPath = "/Time/Simulation Speed",
+  IsLocal = true
+}
+
+local ToggleRotationFriction = {
+  Identifier = "os.ToggleRotationFriction",
+  Name = "Toggle rotation friction",
+  Command = propertyHelper.invert("NavigationHandler.OrbitalNavigator.Friction.RotationalFriction"),
+  Documentation = "Toggles the rotational friction of the camera. If it is disabled, the camera rotates around the focus object indefinitely",
+  GuiPath = "/Navigation",
+  IsLocal = false
+}
+
+local ToggleZoomFriction = {
+  Identifier = "os.ToggleZoomFriction",
+  Name = "Toggle zoom friction",
+  Command = propertyHelper.invert("NavigationHandler.OrbitalNavigator.Friction.ZoomFriction"),
+  Documentation = "Toggles the zoom friction of the camera. If it is disabled, the camera rises up from or closes in towards the focus object indefinitely",
+  GuiPath = "/Navigation",
+  IsLocal = false
+}
+
+local ToggleRollFriction = {
+  Identifier = "os.ToggleRollFriction",
+  Name = "Toggle roll friction",
+  Command = propertyHelper.invert("NavigationHandler.OrbitalNavigator.Friction.RollFriction"),
+  Documentation = "Toggles the roll friction of the camera. If it is disabled, the camera rolls around its own axis indefinitely",
+  GuiPath = "/Navigation",
+  IsLocal = false
+}
+
+local FadeToBlack = {
+  Identifier = "os.FadeToBlack",
+  Name = "Fade to/from black",
+  Command = [[
+    if openspace.getPropertyValue("RenderEngine.BlackoutFactor") > 0.5 then
```

Comment:
Maybe not part of the coding style, but something I've noticed that we do a bit all over. 

For scripts like these, should we remove the extra indentation? I guess to not print it in the script log as well